app/controllers/api/v1/dishes_controller.rb

```diff
@@ -0,0 +1,11 @@
+class Api::V1::DishesController < ApplicationController
+  def index
+    render json: Dish.all
+  end
+
+  def show
+    dish = Dish.find(params[:id])
+    reviews = Review.where(dish_id: params[:id])
+    render json: reviews
+  end
+end
```

Comment:
So excited to see you use this namespaced API pattern. With versioning, even! Yes.